### PR TITLE
add Accept-Language header and builder method to be able to get local…

### PR DESCRIPTION
**Pull request by allanman**
…ized data

### CODE REVIEW

**Comment by itstexter** [comment]
Looks good! 

If you could sign the [Uber CLA](https://docs.google.com/a/uber.com/forms/d/1pAwS_-dA1KhPlfxzYLBqK6rsSWwRwH95OCCZrcsY5rk/viewform) I can get this merged!

**Comment by allanman** [comment]
done!

**Comment by allanman** [comment]
Please have a look, see if you think that covers it.
Accepted the CLA.

File: sdk/src/main/java/com/uber/sdk/rides/client/Session.java [hunk 1]
```diff
@@ -96,6 +100,15 @@ public Builder setServerToken(String serverToken) {
             this.serverToken = serverToken;
             return this;
         }
+        
+        /**
+         * Sets the requested locale through the Accept-Language http header. See https://developer.uber.com/docs/localization for
```
Comment (alecmgo) [inline sdk/src/main/java/com/uber/sdk/rides/client/Session.java]: Nit: End sentence with a period, linkify the URL, capitalize HTTP, lowercase Locales.

"Sets locale in the Accept-Language HTTP header. See the <a href="https://developer.uber.com/docs/localization">Uber API Localization documentation</a> for possible locales."

File: sdk/src/main/java/com/uber/sdk/rides/client/Session.java [hunk 2]
```diff
@@ -139,4 +152,12 @@ public Credential getCredential() {
     public String getServerToken() {
         return serverToken;
     }
+    
+    /**
+     * Get the requested language locale for API requests
```
Comment (alecmgo) [inline sdk/src/main/java/com/uber/sdk/rides/client/Session.java]: Nit: End in period.

File: sdk/src/main/java/com/uber/sdk/rides/client/Session.java [hunk 3]
```diff
@@ -96,6 +100,15 @@ public Builder setServerToken(String serverToken) {
             this.serverToken = serverToken;
             return this;
         }
+        
+        /**
+         * Sets the requested locale through the Accept-Language http header. See https://developer.uber.com/docs/localization for
+         * possible Locales
+         */
+        public Builder setAcceptLanguage(Locale l) {
```
Comment (itstexter) [inline sdk/src/main/java/com/uber/sdk/rides/client/Session.java]: spell out locale as the variable here i.e. setAcceptLanguage(Locale locale)

**Comment by itstexter** [comment]
Thanks so much @allanman!

**Pull request merged**